ci/autoformat.sh

```diff
@@ -11,4 +11,4 @@ do
 done
 
 # Cpplint
-cpplint --counting=detailed --recursive common/ desktop_projects/ stm32_projects/device_drivers/ platformio_projects/src/ platformio_projects/lib/device_drivers/ platformio_projects/lib/new_hal/
+cpplint --counting=detailed --filter="-build/c++11" --recursive common/ desktop_projects/ stm32_projects/device_drivers/ platformio_projects/src/ platformio_projects/lib/device_drivers/ platformio_projects/lib/new_hal/
```

Comment:
What issue does this fix? Assuming some warning we don't want?

Reply:
cpplint is stuck on C++11, so we have to suppress or just dont include crono, ever